app/views/static/auth_key_pair_cloud.html.haml

```diff
@@ -0,0 +1,57 @@
+%h3
```

Comment:
This is just moved template from app/views/auth_key_pair_cloud -> app/views/static